consumer/pdata/trace_test.go

```diff
@@ -18,7 +18,7 @@ import (
 	"testing"
 
 	gogoproto "github.com/gogo/protobuf/proto"
-	goproto "github.com/golang/protobuf/proto"
+	goproto "github.com/golang/protobuf/proto" //lint:ignore SA1019 golang/protobuf/proto is deprecated
```

Comment:
Can't replace it with `"google.golang.org/protobuf/proto"` because the OTLP types generated by `gogo/protobuf` don't implement `ProtoReflect()` function of `ProtoMessage` interface:
- https://pkg.go.dev/google.golang.org/protobuf@v1.25.0/proto?tab=doc#Marshal
- https://pkg.go.dev/google.golang.org/protobuf/reflect/protoreflect?tab=doc#ProtoMessage